Clarify requirements (#8)
freight-hub/freighthub-frontend-challenge: README.md

# How to submit

- Clone this repository.
- A RESTful API for `shipments` is provided with the challenge. To run, follow: [How to run API server](#how-to-run-api-server)
- Complete your project as described above within your local repository.
- Make sure that there are scripts to start both the server and the client.

> - Ensure everything you want to commit is committed (or else your code will not be included the bundle).

Reviewer: ```suggestion
- Ensure everything you want to commit is committed before you bundle
```
